### ISSUE TITLE
Still 1.3.17 version, even if 1.3.18 is ready. Wrong installation by bower. 

### ISSUE THREAD

**Issue by Deliaz** [marcuswestin/store.js#141]
Hello,
There is a problem: bower installs wrong package version. 
After adding UMD pattern (7c65e0a) there wasn't any version changes (except chenglog). 
Any plans? 

Pull request https://github.com/marcuswestin/store.js/pull/127 & https://github.com/marcuswestin/store.js/pull/123
Related https://github.com/marcuswestin/store.js/issues/139

For those, who looking current version with UMD: 
```
bower install https://github.com/marcuswestin/store.js.git#master
```

**Comment by purtuga** [comment]
+1

**Comment by marcuswestin** [comment]
Fixed with v1.3.19 in c6c006eee1811.

I've been gone a long time. Really not appropriate as a maintainer. My bad guys, sorry for the long wait, thanks for all the patience, and I hope you'll all find good use of upcoming changes! (See https://github.com/marcuswestin/store.js/blob/master/Roadmap.md for details)
